docs/metrics.md

```diff
@@ -70,6 +70,7 @@ overlay.async.read                       | meter     | number of async read requ
 overlay.async.write                      | meter     | number of async write requests issued
 overlay.connection.authenticated         | counter   | number of authenticated peers
 overlay.flow-control.percentage          | counter   | percentage of authenticated connections that enable flow control
+overlay.pull-mode.percentage             | counter   | percentage of authenticated connections that enable pull mode
```

Comment:
actually before you refresh those metrics let's talk: there might be something wrong